kuma/conftest.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from django.conf import settings
+from django.core.cache import caches
+from django.utils.translation import trans_real
+
+from kuma.core.tests import SessionAwareClient, LocalizingClient
+
+
+@pytest.fixture
+def db_and_empty_caches(db):
+    """
+    Use this fixture to emulate KumaTestCase. In addition to the
+    standard set-up and tear-down that the "db" fixture provides
+    (to emulate django.test.TestCase), it will clear the caches
+    before (but not after) the test is run.
+    """
+    for cache in caches.all():
+        cache.clear()
+
+    trans_real.deactivate()
+    trans_real._translations.clear()  # Django fails to clear this cache.
+    trans_real.activate(settings.LANGUAGE_CODE)
+
+
+@pytest.fixture
+def session_aware_client():
+    """
+    Provides a session-aware client.
+    """
+    return SessionAwareClient()
+
+
+@pytest.fixture
+def localizing_client():
```

Comment:
Again, unused in any current tests, and I'd like to wait until needed to add it.  This client works with our unique localizing scheme.  I hope to evaluate getting back to [built-in Django techniques](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#django.conf.urls.i18n.i18n_patterns) in the near future.